crates/subspace-networking/src/constructor.rs

```diff
@@ -296,7 +303,10 @@ where
             .set_replication_interval(None);
 
         let mut yamux_config = YamuxConfig::default();
-        yamux_config.set_max_num_streams(YAMUX_MAX_STREAMS);
+        yamux_config
+            .set_max_num_streams(YAMUX_MAX_STREAMS)
+            .set_receive_window_size(YAMUX_RECEIVING_WINDOW as u32)
+            .set_max_buffer_size(YAMUX_BUFFER_SIZE);
```

Comment:
Just a nit, but I think it speaks about attention to details.

We have 3 constants related to Yamux that are used one after another. Yet:
1) You have added new constants into a completely different place than constant that existed previously
2) Even those two new constants are out of order comparing to how they are used (and this is the only place where they are used)

Why not grouping all 3 together in the same exact order as they are used here? Just feels like introducing unnecessary entropy.

Reply:
It makes the changes less clear but it's no problem.